manifests/config.pp

```diff
@@ -97,4 +97,22 @@
       refreshonly => true,
     }
   }
+
+  if $pulp::lazy_enabled {
+    class { '::squid3':
```

Comment:
its not the only supported option, any http proxy would work i would imagine, but they provide same configs for two different proxies, squid and one more that isn't in the base OS.